core/domain/rating_services.py

```diff
@@ -71,6 +71,12 @@ def _update_user_rating():
     exploration_summary.ratings[str(new_rating)] += 1
     if old_rating:
         exploration_summary.ratings[str(old_rating)] -= 1
+
+    calculated_scaled_average_rating = (
```

Comment:
Not sure what "calculated" adds here. Why not just call this get_scaled_average_rating()?